source/manual/pact-broker.html.md

```diff
@@ -76,3 +76,7 @@ Follow these steps in order to change the provider and consumer in tandem.
 
 1. Re-run the build for the provider PR now the consumer is merged.
   - The build should pass so you can now merge the PR.
+
+## Special cases and tech debt
+
+Publishing API and Content Store have a direct pact, with [Publishing API acting as the consumer](https://github.com/alphagov/publishing-api/tree/dd8dd9232d3cbf33b8945fdd898ebe80d7dcfcf6/spec/pacts/content_store) and [Content Store acting as the provider](https://github.com/alphagov/content-store/blob/de729dfe12e6e9da4a27a52259f59b9051e4da27/spec/service_consumers/pact_helper.rb#L32). This can be confusing as [Publishing API is also a provider for GDS API Adapters](https://github.com/alphagov/publishing-api/blob/dd8dd9232d3cbf33b8945fdd898ebe80d7dcfcf6/spec/service_consumers/pact_helper.rb#L20). It's unclear if the direct pact was intentional. In future we should consider changing Publishing API to use GDS API Adapters to talk to Content Store and have an indirect pact like we do for all other apps.
```

Comment:
Hmm...you make a fair point that my claim of "tech debt" is unsubstantiated (it's becoming a habit!). I think there is _some_ debt here: the inconsistency isn't necessarily an issue, but it's undocumented.

However, I also agree that this might be the preferred way of implementing one-to-one contract tests. Content Data takes a similar approach, but doesn't have any contract tests to make this more than a one-off.

I'm open to removing "and tech debt" and cutting from "It's unclear...". Just before I do, I'd like to ask @kevindew for a third opinion: does the current approach surprise you?

Reply:
For the sake of getting the bulk of this change in, I've cut the contentious part of this and created [a tech debt card](https://trello.com/c/GkIffOx8/232-publishing-api-contract-tests-are-different-to-all-other-repos) to discuss adding more guidance on this special case.

@ChrisBAshton just a final check: good to go?

Reply:
LGTM 👍